config/main.py

```diff
@@ -4970,6 +4974,12 @@ def del_vrf(ctx, vrf_name):
         ctx.fail("'vrf_name' is not start with Vrf, mgmt or management!")
     if len(vrf_name) > 15:
         ctx.fail("'vrf_name' is too long!")
+    syslog_table = config_db.get_table("SYSLOG_SERVER")
+    syslog_vrf_dev = "mgmt" if vrf_name == "management" else vrf_name
```

Comment:
>vrf_name [](http://example.com/codeflow?start=31&length=8)

@prsunny, Could you check VRF related part?

Reply:
lgtm, minor comment: Later part of the syslog implementation checks for the presence of 'source_ip' in linux during add operation. But this is not checked during an `ip address` delete. Is this intentional or am i missing something?

Reply:
@prsunny you are right, this is intentionally. The reason for that is because we don't reference IP in Config DB, so there is no point to check

Reply:
@prsunny can you please approve if no more questions